### PR TITLE
Fix bad slice numbering

### LINKED ISSUE #200: cubeslice from-to behaviour is not as expected
- would expect [FROM:TO] but if FROM==TO then no output is made. if they differ then [FROM:TO[
- also counting the slices outputted starts at zero (0/102) and terminates one index earlier than expected (101/102)

### PR DESCRIPTION
**Pull request by nbassler**
Closes #200

### CODE REVIEW

File: pytrip/utils/cubeslice.py [hunk 1]
```diff
@@ -200,6 +200,10 @@ def main(args=sys.argv[1:]):
     ct_im = None
     data_im = None
 
+    if args.sstart == 0:
```
Comment (grzanka) [inline pytrip/utils/cubeslice.py]: Isn't it better to check if `args.sstart < 1` ?

File: pytrip/utils/cubeslice.py [hunk 2]
```diff
@@ -200,6 +200,10 @@ def main(args=sys.argv[1:]):
...
+        logger.error("First slice must be 1 or greater.")
```
Comment (grzanka) [inline pytrip/utils/cubeslice.py]: I'd recommend also to provide an information about value provided by user, i.e.:
"you provided XXXX, but first slice must be 1 or greater"

File: pytrip/utils/cubeslice.py [hunk 3]
```diff
@@ -227,20 +231,31 @@ def main(args=sys.argv[1:]):
...
             # use os.path.join() in order to add slash if it is missing.
             output_filename = os.path.join(args.outputdir, os.path.basename(output_filename))
         if args.verbosity == 0:
-            print("Write slice number: " + str(slice_id) + "/" + str(cube.dimz))
+            print("Write slice number: " + slice_str)
         if args.verbosity > 0:
-            logger.info("Write slice number: " + str(slice_id) + "/" + str(cube.dimz) + " to " + output_filename)
+            logger.info("Write slice number: " + slice_str + " to " + output_filename)
+
+        if ct_cube is not None:
```
Comment (grzanka) [inline pytrip/utils/cubeslice.py]: You can reduce this code to:
```slice_str += " zpos = {:.2f} mm".format(cube.slice_to_z(slice_id - 1))```
We have `cube` variable which points to `data_cube` or, if its missing to `ct_cube`.
See line 168

File: pytrip/utils/cubeslice.py [hunk 4]
```diff
@@ -227,20 +231,31 @@ def main(args=sys.argv[1:]):
     ax.xaxis.set_minor_locator(minorLocator)
     ax.yaxis.set_minor_locator(minorLocator)
 
+    text_slice = ax.text(0.0, 1.05, "", size=8, transform=ax.transAxes)
```
Comment (grzanka) [inline pytrip/utils/cubeslice.py]: Why not to use plot title to hold such information ?

Comment (nbassler) [inline pytrip/utils/cubeslice.py]: I might want to move it later if more info is added. In fact id like smth like the pytripgui rendering. But this really not important right now..

**Comment by grzanka** [comment]
Here https://github.com/pytrip/pytrip/blob/master/tests/test_converters.py#L269 test expects 1 file, but after your change two are generate. Fix test please

**Pull request merged**